README.md

```diff
@@ -167,4 +169,18 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
 
 You can add the sign off when creating the git commit via `git commit -s`.
 
+### Commit Style
+
+Simple house-keeping for clean git history.
+Read more on [How to Write a Git Commit Message](http://chris.beams.io/posts/git-commit/) or the Discussion section of [`git-commit(1)`](http://git-scm.com/docs/git-commit).
+
+1. Separate subject from body with a blank line
```

Comment:
“Separate subject” → “Separate the subject”.  I know you're just echoing Chris' blog here, but all of our other “subject” occurrences have an article.